forte/data/extractor/char_extractor.py

```diff
@@ -30,7 +30,9 @@
 
```

Comment:
I notice you import `Annotation` from `base_ontology`, but you should really import from `data.ontology.top`